app/helpers/general-date.js

```diff
@@ -1,14 +1,22 @@
 import Helper from '@ember/component/helper';
 import moment from 'moment';
 
+const dateFormats = {
+  'date-time-long'     : 'dddd, D MMMM, YYYY h:mm A',
+  'date-time-tz-long'  : 'dddd, D MMMM, YYYY h:mm A (z)',
+  'date-time-short'    : 'D MMM, YYYY h:mm A',
+  'date-time-tz-short' : 'D MMM, YYYY h:mm A (z)',
+  'date-short'         : 'D MMM, YYYY'
+};
+
 const locales12Hours = new Set(['en', 'bn', 'hi', 'id', 'ja', 'run', 'th', 'vi', 'ko']);
 
 export function generalDate(params, { tz }) {
   const timezone = tz || moment.tz.guess();
 
   const local = moment(params[0]).tz(timezone).locale();
 
-  let format = params[1] || 'h:mm A, MMMM Do YYYY (z)';
+  let format = (dateFormats[params[1]] || params[1]) || 'h:mm A, MMMM Do YYYY (z)';
```

Comment:
Should I handle all the cases bcz at some places format is being used only 2-3 times and so I provided this here 
```js
let format = (dateFormats[params[1]] || params[1]) || 'h:mm A, MMMM Do YYYY (z)';
```
This will handle all the cases where key-word is passed or format is passed

Reply:
Like in some places there is 'YYYY' or 'h: mm A' or 'HH: mm' so either I have to make a key-word for them as well.
pls let me know

Reply:
OK, leave them if they don't contain timezone

Reply:
Most of them don't contain, 1-2 contain but that will handled by general-date

Reply:
Pls let me know to change anything else

Reply:
It can't handle for #5798 

Which ones contain timezone, please write here

Reply:
Should I fix #5798 here as well? bcz there is a PR for that before?

Reply:
Only at one place 'h: mm A (z)' in public/session.hbs

Reply:
Can simple date and cell-date use general-date helper? And then we make this format `time-tz`?

We need it for #5798 

And no, we don't need to handle the above issue in this PR

Reply:
yes we can do that

Reply:
@iamareebjamal Done, it was not in cell-date, only in cell-simple-date